modules/og_context/src/Plugin/OgContext/Enitty.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Drupal\og_context\Plugin\OgContext;
+
+use Drupal\og_context\Plugin\OgContextBase;
+
+/**
+ * @OgContext(
+ *  id = "entity",
+ *  title = "Entity base",
+ *  description = @Translation("Get the group from the current entity.")
```

Comment:
`Get the group from the current entity, by checking if it is a group or a group content entity.`